src/aiortc/contrib/media.py

```diff
@@ -317,6 +349,7 @@ def _start(self, track: PlayerStreamTrack) -> None:
             self.__thread.start()
 
     def _stop(self, track: PlayerStreamTrack) -> None:
+        print("Stopping in MediaPlayer", track.kind, track)
```

Comment:
is this deliberately a print statement? can we just log this?

Reply:
All print statements were to debug the weird bug in which the code stops at frame 12-15 in bilayer and 38-40 in FOM